src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/advice/StaffReferenceException.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.advice;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public class StaffReferenceException extends RuntimeException {
+
+    private final HttpStatus status;
+    private final String errorMessage;
+    private final Throwable exception;
+
+    public StaffReferenceException(HttpStatus status,
+                                   String errorMessage,
+                                   Throwable exception) {
+        super(errorMessage, exception);
+        this.status = status;
```

Comment:
Trivial comment just overload another constructor  public StaffReferenceException(
                                   String errorMessage,
                                   Throwable exception) 
hence we can use it without HTTP status also

Reply:
sure. Will do